lib/net.js

```diff
@@ -1041,6 +1048,73 @@ function internalConnect(
 }
 
 
+function internalConnectMultiple(context) {
+  clearTimeout(context[kTimeout]);
+  const self = context.socket;
+  assert(self.connecting);
+
+  // All connections have been tried without success, destroy with error
+  if (context.current === context.addresses.length) {
```

Comment:
`ArrayPrototypeLength(context.addresses)`

Reply:
`primordials.ArrayPrototypeLength` is not a function, it's the number `0` – and it's really not a useful primordials, it's there only because `Array.isArray(Array.prototype)` is true. FWIW `length` is an own property of arrays, not a getter defined on their prototype.